File: 1480/main.cpp
#include "bits/stdc++.h"
using namespace std;

class Solution {
public:
    void runningSum(vector<int>& nums) {
        int store = 0;
        vector<int> ans (nums.size());

        for (size_t i = 0; i < nums.size(); i++){
            store = nums[i] + store;
            ans.push_back(store);
        }
    }
};

int main() {
    Solution sol;
    int n;
    cin >> n;
    vector<int> nums(n);
    for (int i = 0; i < n; i++){
        cin >> nums[i];
    }
    sol.runningSum(nums);
    return 0;
}

File: 1480/main.py
from typing import List

class Solution(object):
    def runningSum(self, nums: List[int]) -> List[int]:
        ans = []
        store = 0
        for i in nums:
            store = i + store
            ans.append(store)
        return ans


if __name__ == "__main__":
    sol = Solution()
    print(sol.runningSum([1,2,3,4]))  # Expected: